conf/schemas.go

```diff
@@ -116,6 +117,19 @@ func ReadSchemas(file string) (Schemas, error) {
 		}
 		schema.Priority = int64(p)<<32 - int64(i) // to sort records with same priority by position in file
 
+		reorderBufferStr := sec.ValueOf("reorderBuffer")
+		if len(reorderBufferStr) > 0 {
+			reorderWindow, err := strconv.ParseUint(reorderBufferStr, 10, 32)
```

Comment:
actually it's the same for the two `storage-schemas.conf` in:
`docker/docker-dev-custom-cfg-kafka/storage-schemas.conf`
`docker/docker-cluster/storage-schemas.conf`

Should i copy the comment from `scripts/config/storage-schemas.conf` to all of them while i'm at it?